### PR TITLE
remove preload

### PR DESCRIPTION
**Pull request by hayarobi**
This PR reduces code complexity by sacrificing approximately 10% of performance.

### CODE REVIEW

File: contract/contract.go [hunk 1]
```diff
@@ -13,61 +14,30 @@ import (
...
-	loadReqCh     chan *preloadRequest
-	preloaders    [2]preloader
 	PubNet        bool
 	TraceBlockNo  uint64
 	bpTimeout     <-chan struct{}
 	maxSQLDBSize  uint64
 	addressRegexp *regexp.Regexp
 )
 
+// These constants indicate the situation in which the contract is executed. BlockFactory refers to execution by a block producer to create a block, from which BlockFactory class calls, and ChainService refers to execution to verify and reflect blocks created by other producers. Depending on the mode, the operations or policies may be different, and currently timeout policy is different.
```
Comment (kroggen) [inline contract/contract.go:27]: reflect -> apply

File: contract/vm.go [hunk 2]
```diff
@@ -194,7 +194,7 @@ func NewVmContext(execCtx context.Context, blockState *state.BlockState, cdb Cha
 		confirmed:       confirmed,
 		isQuery:         query,
 		blockInfo:       bi,
-		service:         C.int(service),
+		service:         C.int(executionMode),
```
Comment (kroggen) [inline contract/vm.go:197]: maybe later the `service` from `vmContext` can also be renamed `executionMode`

File: chain/chainhandle.go [hunk 3]
```diff
@@ -653,22 +653,14 @@ func (e *blockExecutor) execute() error {
...
 		defer contract.CloseDatabase()
-		var preloadTx *types.Tx
-		numTxs := len(e.txs)
-		for i, tx := range e.txs {
-			// if tx is not the last one, preload the next tx
-			if i != numTxs-1 {
-				preloadTx = e.txs[i+1]
-				contract.RequestPreload(e.BlockState, e.bi, preloadTx, tx, contract.ChainService)
-			}
+		logger.Trace().Int("txCount", len(e.txs)).Msg("executing txs (maybe for connecting block)")
```
Comment (kroggen) [inline chain/chainhandle.go:656]: printing "maybe for ..." is not useful information. that part can be removed

File: contract/contract.go [hunk 4]
```diff
@@ -13,61 +14,30 @@ import (
...
+// TODO These values are also used to select slots for internal parallel processing. They have multiple roles or meanings so that mit make harder to understand the source code.
 const (
 	BlockFactory = iota
 	ChainService
-	MaxVmService
+	MaxVmService // MaxVmService
```
Comment (kroggen) [inline contract/contract.go:32]: there is no need to repeat the constant name as a comment

File: contract/vm.go (hunk 2 above)
Comment (hayarobi) [reply contract/vm.go:197]: The field `service` seems to have multiple meaings as commented in contract.go file above. IMO, it is better to refactor the overall logic later time.

File: contract/contract.go (hunk 4 above)
Comment (gokch) [reply contract/contract.go]: it seems MaxVmService no more need.

Comment (hayarobi) [reply contract/contract.go]: This symbol will be removed when the entire logic is refactored.

**Review by kroggen — APPROVED** [review]

**Comment by kslee8282** [comment]
sync mainnet
Dec  7 05:26:30.584 INF ../go/aergo/cmd/aergosvr/aergosvr.go:98 > AERGO SVR STARTED branch=HEAD module=asvr revision=41c26774

**Comment by kslee8282** [comment]
sync success

**Pull request merged**